villas/controller/components/managers/kubernetes_simple.py

```diff
@@ -0,0 +1,82 @@
+from villas.controller.components.managers.kubernetes import KubernetesManager
+from villas.controller.components.simulators.kubernetes import KubernetesJob
+
+parameters_simple = {
```

Comment:
Please turn this into a function which constructs the dictionary and returns it.

The patching of the struct below is not nice, since it modifies the global variable.